go/engine/context.go

```diff
@@ -21,6 +23,7 @@ type Context struct {
 	UpdateUI    libkb.UpdateUI
 
 	LoginContext libkb.LoginContext
+	NContext     context.Context
```

Comment:
Fine.  I changed to GetNetContext().  Hopefully all the names will be cleaned up when we consolidate all the various contexts.